tools/clang/unittests/HLSLExec/ShaderOpArithTable.xml

```diff
@@ -7524,4 +7524,308 @@
             </Parameter>
         </Row>
     </Table>
+    <Table Id="LongVector_BinaryOpTable">
+    <ParameterTypes>
+        <ParameterType Name="ShaderOp.Target">String</ParameterType>
+        <ParameterType Name="ShaderOp.Arguments">String</ParameterType>
+        <ParameterType Name="ShaderOp.Text">String</ParameterType>
+        <ParameterType Name="Validation.Tolerance">double</ParameterType>
+        <ParameterType Array="true" Name="Validation.VecInput1">String</ParameterType>
+        <ParameterType Array="true" Name="Validation.VecInput2">String</ParameterType>
+        <ParameterType Array="true" Name="Validation.ScalarInput">String</ParameterType>
+        <ParameterType Array="true" Name="Validation.VecExpected">String</ParameterType>
+    </ParameterTypes>
+    <Row Name="LongVector_Scalar_Multiply">
+        <Parameter Name="Validation.Tolerance">0</Parameter>
+        <Parameter Name="ShaderOp.Text">
+            <![CDATA[
+
+            template<typename T>
+            struct SLongVectorBinaryOp{
+                T scalarInput;
+                vector<T, NUM> vecInput1;
+                vector<T, NUM> vecInput2;
+                vector<T, NUM> vecOutput;
+            };
```

Comment:
Wondering why `T` as a template parameter (instead of just using `TYPE`), but using the `NUM` macro.

Reply:
Good point, the template isn't needed. I'll update to just use macro values.